### ISSUE TITLE
Installation stop due to the error "Error in install/bootstrap-snuba.sh:3. '$dcr snuba-api bootstrap --no-migrate --force' exited with status 1"

### ISSUE THREAD

**Issue by vsfomin** [getsentry/self-hosted#2063]
### Self-Hosted Version

23.3.1

### CPU Architecture

x86_64

### Docker Version

23.0.2

### Docker Compose Version

2.17.2

### Steps to Reproduce

1. Clone repo git clone --depth 1 --branch 23.3.1 https://github.com/getsentry/self-hosted.git
2. run ./install.sh
3. Get an error:
```
Error in install/bootstrap-snuba.sh:3.
'$dcr snuba-api bootstrap --no-migrate --force' exited with status 1
-> ./install.sh:main:31
--> install/bootstrap-snuba.sh:source:3

```

### Expected Result

Successfully running containers

### Actual Result

Logs from file sentry_install_log-2023-04-03_16-43-31.txt:

> Unable to find image 'curlimages/curl:7.77.0' locally
> 7.77.0: Pulling from curlimages/curl
> 339de151aab4: Pulling fs layer
> 9576f9a419e9: Pulling fs layer
> 5b67e0e9bce9: Pulling fs layer
> 7c0640a6c519: Pulling fs layer
> 1d8f7d50664a: Pulling fs layer
> 3ce74da83aeb: Pulling fs layer
> 7094fabf5e16: Pulling fs layer
> 2f4d0b649c1f: Pulling fs layer
> 110e7f874674: Pulling fs layer
> 1d8f7d50664a: Waiting
> 3ce74da83aeb: Waiting
> 7094fabf5e16: Waiting
> 2f4d0b649c1f: Waiting
> 110e7f874674: Waiting
> 7c0640a6c519: Waiting
> 5b67e0e9bce9: Verifying Checksum
> 5b67e0e9bce9: Download complete
> 339de151aab4: Verifying Checksum
> 339de151aab4: Download complete
> 339de151aab4: Pull complete
> 9576f9a419e9: Verifying Checksum
> 9576f9a419e9: Pull complete
> 5b67e0e9bce9: Pull complete
> 7c0640a6c519: Download complete
> 7c0640a6c519: Pull complete
> 1d8f7d50664a: Verifying Checksum
> 1d8f7d50664a: Download complete
> 1d8f7d50664a: Pull complete
> 3ce74da83aeb: Download complete
> 3ce74da83aeb: Pull complete
> 2f4d0b649c1f: Download complete
> 7094fabf5e16: Download complete
> 7094fabf5e16: Pull complete
> 2f4d0b649c1f: Pull complete
> 110e7f874674: Verifying Checksum
> 110e7f874674: Download complete
> 110e7f874674: Pull complete
> Digest: sha256:6e0a786e3e5181df00eaf3a0a1749c18a6bb20b01c9bd192ea72176ce8a1c94b
> Status: Downloaded newer image for curlimages/curl:7.77.0
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
>   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
> 100  101k  100  101k    0     0   143k      0 --:--:-- --:--:-- --:--:-- 7783k
> 
> ▶ Bootstrapping and migrating Snuba ...
>  Container sentry-self-hosted-redis-1  Creating
>  Container sentry-self-hosted-clickhouse-1  Creating
>  Container sentry-self-hosted-zookeeper-1  Creating
>  Container sentry-self-hosted-redis-1  Created
>  Container sentry-self-hosted-clickhouse-1  Created
>  Container sentry-self-hosted-zookeeper-1  Created
>  Container sentry-self-hosted-kafka-1  Creating
>  Container sentry-self-hosted-kafka-1  Created
>  Container sentry-self-hosted-zookeeper-1  Starting
>  Container sentry-self-hosted-redis-1  Starting
>  Container sentry-self-hosted-clickhouse-1  Starting
>  Container sentry-self-hosted-redis-1  Started
>  Container sentry-self-hosted-zookeeper-1  Started
>  Container sentry-self-hosted-zookeeper-1  Waiting
>  Container sentry-self-hosted-clickhouse-1  Started
>  Container sentry-self-hosted-zookeeper-1  Error
> dependency failed to start: container sentry-self-hosted-zookeeper-1 is unhealthy
> Error in install/bootstrap-snuba.sh:3.
> '$dcr snuba-api bootstrap --no-migrate --force' exited with status 1
> -> ./install.sh:main:31
> --> install/bootstrap-snuba.sh:source:3
> 
> Looks like you've already sent this error to us, we're on it :)

**Comment by Milksystems** [comment]
dependency failed to start: container sentry-self-hosted-kafka-1 is unhealthy
Error in install/bootstrap-snuba.sh:3.
'$dcr snuba-api bootstrap --no-migrate --force' exited with status 1
-> ./install.sh:main:31
--> install/bootstrap-snuba.sh:source:3

Looks like you've already sent this error to us, we're on it :)

Docker version 23.0.2
docker-compose version 1.29.2

**Comment by vsfomin** [comment]
zookeeper containber logs:
===> Launching zookeeper ...
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f4e9be2c532, pid=1, tid=0x00007f4e9c9ee700
#
# JRE version: OpenJDK Runtime Environment (Zulu 8.38.0.13-CA-linux64) (8.0_212-b04) (build 1.8.0_212-b04)
# Java VM: OpenJDK 64-Bit Server VM (25.212-b04 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x36532]  abort+0x232
#
# Core dump written. Default location: //core or core.1
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid1.log
#
# If you would like to submit a bug report, please visit:
#   http://www.azulsystems.com/support/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

[error occurred during error reporting , id 0xb]


[error occurred during error reporting , id 0xb]

**Comment by vsfomin** [comment]
```
===> Launching zookeeper ...
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f4e9be2c532, pid=1, tid=0x00007f4e9c9ee700
#
# JRE version: OpenJDK Runtime Environment (Zulu 8.38.0.13-CA-linux64) (8.0_212-b04) (build 1.8.0_212-b04)
# Java VM: OpenJDK 64-Bit Server VM (25.212-b04 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x36532]  abort+0x232
#
# Core dump written. Default location: //core or core.1
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid1.log
#
# If you would like to submit a bug report, please visit:
#   http://www.azulsystems.com/support/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

[error occurred during error reporting , id 0xb]


[error occurred during error reporting , id 0xb]
```

**Comment by hubertdeng123** [comment]
yeah, this is a duplicate of [this](https://github.com/getsentry/self-hosted/issues/1445#issuecomment-1122415883)

Seems to be more common now these days, perhaps this should belong in the documentation
